apps/zbugs/shared/auth.ts

```diff
@@ -90,3 +89,9 @@ export async function assertUserCanSeeComment(
 
   await assertUserCanSeeIssue(tx, userID, comment.issueID);
 }
+
+declare module '@rocicorp/zero' {
+  interface DefaultTypes {
+    context: AuthData;
```

Comment:
I would have preferred if we are more explicit here. I do not want us to add `| undefined` by default.